frontend/tsconfig.json

```diff
@@ -16,7 +16,7 @@
     "strict": true,
     "noEmit": true,
     "module": "ESNext",
-    "target": "es6",
+    "target": "es2019",
     "allowJs": true,
     "lib": ["es5", "es6", "dom", "dom.iterable", "es2019"],
```

Comment:
Das war nötig, damit tsc an einer anderen stelle nicht wegen named-capture groups in regex meckert. Ich habe es2019 genommen, weil es auch schon in Z21 genutzt wird